EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Private key deactivation and reactivation
+description: Introduce a new precompiled contract to enable Externally Owned Accounts (EOAs) to deactivate and reactivate their private keys.
+author: Liyi Guo (@colinlyguo)
+discussions-to: <URL>
```

Comment:
Please create a discussions topic in Eth Magicians using the template: https://ethereum-magicians.org/c/eips/5

Reply:
thanks. updated in [314ed2b](https://github.com/ethereum/EIPs/pull/9193/commits/314ed2b98598f5ad51c5c1a4277686ca232c1df9).